### DISCUSSION TITLE
Ability to delete a discussion

### DISCUSSION THREAD

**Discussion by myckhel** [community/community#3544] [Discussions] [answered]
Can the github team develop the ability to delete a discussion after post.

**Comment by PranamBhat** [comment, marked as answer]
Hi @myckhel 

Currently it is not possible. You can write this as a feature request.

Also write over [https://github.community](https://github.community)

**Reply by myckhel** [reply]
okay thanks for your response.

**Reply by PranamBhat** [reply]
@myckhel Kindly mark this as "**Answered**" and leave the discussion. Thank you.
